src/components/EntityStatus/EntityStatus.tsx

```diff
@@ -90,24 +97,51 @@ export function EntityStatus({
                 </span>
             )}
             {(path || name) && (
-                <div className={b('wrapper', {'with-button': hasClipboardButton})}>
+                <div
+                    className={b('wrapper', {
+                        'with-clipboard-button': hasClipboardButton,
+                        'with-info-button': Boolean(infoPopoverContent),
+                    })}
+                >
                     <span className={b('link', {'with-left-trim': withLeftTrim})} title={name}>
                         {renderLink()}
                     </span>
-                    {hasClipboardButton && (
+                    {(hasClipboardButton || infoPopoverContent) && (
                         <div
                             className={b('controls-wrapper', {
-                                visible: clipboardButtonAlwaysVisible,
+                                visible: clipboardButtonAlwaysVisible || infoIconHovered,
                             })}
                         >
-                            <ClipboardButton
-                                text={name}
-                                size="xs"
-                                view="normal"
-                                className={b('clipboard-button', {
-                                    visible: clipboardButtonAlwaysVisible,
-                                })}
-                            />
+                            {infoPopoverContent && (
```

Comment:
Info button always take place in column. It looks not good if column is narrow enough.
<img width="459" alt="Screenshot 2025-03-27 at 10 41 50" src="https://github.com/user-attachments/assets/27a7dc79-a0a7-4b9d-a5d5-fbe9eab6c038" />